docs/usage-as-end-user.md

```diff
@@ -104,6 +104,19 @@ The `cloudControllerManager.featureGates` contains a map of explicitly enabled o
 For production usage it's not recommend to use this field at all as you can enable alpha features or disable beta/stable features, potentially impacting the cluster stability.
 If you don't want to configure anything for the `cloudControllerManager` simply omit the key in the YAML specification.
 
+## WorkerConfig
+
+The worker configuration mainly contains Local SSD interface for the additional volumes attached to GCP worker machines.
+If you attach the disk with `SCRATCH` type, either an `NVMe` interface or a `SCSI` interface must be specified.
+An example `WokerConfig` for the GCP looks as follows:
+
+```yaml
+apiVersion: gcp.provider.extensions.gardener.cloud/v1alpha1
+kind: WorkerConfig
+volume:
+  interface: NVME
+```
+
```

Comment:
Following up the discussion from https://github.com/gardener/gardener-extension-provider-gcp/pull/71#discussion_r414237155 I suggest to document the behaviour here (i.e., to document that it is only allowed to provide this volume interface if only `SCRATCH` data volumes are used).

Reply:
Done